src/cpp/shared/gui/msg.cpp

```diff
@@ -18,7 +18,7 @@ void GUISetIcon(heroWindow* hwnd, int f, char* p) {
 	GUIBroadcastMessage(hwnd, f, GUI_MESSAGE_SET_ICON, (void*)p);
 }
 
-void GUISetText(heroWindow* hwnd, int f, char* p) {
+void GUISetText(heroWindow* hwnd, int f, const char* p) {
 	GUIBroadcastMessage(hwnd, f, GUI_MESSAGE_SET_TEXT, (void*)p);
```

Comment:
This might be undefined behavior now.  I have to look whether going from const char * to plain void * is undefined.  It appears to work, but still would rather avoid UB if possible.